stl/inc/yvals_core.h

```diff
@@ -1003,12 +1004,23 @@
 #define __cpp_lib_to_array                201907L
 #define __cpp_lib_type_identity           201806L
 #define __cpp_lib_unwrap_ref              201811L
+
+#ifdef __cpp_lib_is_constant_evaluated
+#define __cpp_lib_constexpr_numeric 201911L

```

Comment:
It's acceptable that this isn't in sorted order, due to the ordering dependency, and very soon we'll be able to remove the conditional logic here. Ok as-is.